aws/cloudformation/components/codeprojects.yml.erb

```
@@ -0,0 +1,73 @@
+<%# Note this file must be indented one level, as it is a child of 'Resources'. %>
+  # -----------------------------------------------
+  # Begin codeprojects.org resources
+  # 
+  # This template component includes resources for the codeprojects.org
+  # infrastructure. Currently this is mostly manually deployed.
+  # 
+  # -----------------------------------------------
+
+<%-
+  domain = "codeprojects.org"
+
+  hosted_zone = Aws::Route53::Client.new.list_hosted_zones_by_name(dns_name: domain).hosted_zones.first
+  raise "Could not find #{domain} in hosted zones" unless hosted_zone.name.delete_suffix('.') == domain
+  hosted_zone_id = hosted_zone.id.delete_prefix("/hostedzone/")
+
+  envs = {
+    staging: {
+      domain_name: "staging.#{domain}",
+      alternative_names: []
+    },
+    # test: {
+    #   domain_name: "test.#{domain}",
+    #   alternative_names: []
+    # },
+    # production: {
+    #   domain_name: domain,
+    #   alternative_names: ["www.#{domain}"]
+    # },
+  }
+-%>
+
+<%- if rack_env?(:staging) -%>
+  # Developer Resources (Deployed with Staging)
+  LocalhostCodeprojectsRecord:
+    Type: AWS::Route53::RecordSet
+    Properties:
+      HostedZoneId: <%=domain%>
+      Name: <%="localhost.#{domain}"%>
+      Type: A
+      TTL: 3600
+      ResourceRecords:
+        - 127.0.0.1
+<%- end -%>
+
+<%- envs.each do |env, config| -%>
+<%- if rack_env == env -%>
```

any reason not to use 

```suggestion
<%- if rack_env?(env) -%>
```

here, too?

---

I don't fully understand the difference between `rack_env?(:production)` and `rack_env == "production"`, and erred on the side that I knew would work.

If your suggestion is syntactically valid and idiomatic, then I'm all for it.